### ISSUE TITLE
error: Name "capsule" is not defined

### ISSUE THREAD

**Issue by mdorier** [sizmailov/pybind11-stubgen#86]
I have a pybind11-based library that uses capsules (`pybind11::capsule`). When I try to use the stubs generated by pybind11-stubgen with mypy, I'm getting a lot of errors like the following:

```
stubs/_pymargo-stubs/__init__.pyi:318: error: Name "capsule" is not defined
```

This can be fixed by manually adding the "capsule" class to __init__.pyi, but it would be better if pybind11-stubgen could do it by itself.

**Comment by sizmailov** [comment]
Hi, thanks for the report.

`PyCapsule` does not provide any _python_ interface to interact with, therefore it would be perfectly fine to annotate it as `Any`
```python
_my_capsule: typing.Any  # PyCapsule()
```
... or even remove it from the stubs altogether (since you can't do much with it from python). 

I'm inclined to implement the first option.

**Comment by sizmailov** [comment]
Closed by 106712441fd5ce4d6a6130d7c52d5b9160ac0738

**Comment by mdorier** [comment]
The problem appears when a C++ function takes a capsule as argument or returns a capsule. pybind11-stubgen will generate a stub that uses a `capsule` type that isn't defined anywhere. The solution I chose was to manually add `class capsule: pass` in the generated stubs, but I guess `capsule = typing.Any` would work as well.

I fail to see how defining `_my_capsule: typing.Any` would solve the problem...
